rpc/grpc/server/src/connection.rs

```diff
@@ -121,11 +146,19 @@ impl Router {
             return Err(GrpcServerError::InvalidRequestPayload);
         }
         let rpc_op = request.payload.as_ref().unwrap().into();
+        let method = self.interface.get_method(&rpc_op);
         let sender = self.routing_map.get(&rpc_op).cloned();
         let sender = sender.unwrap_or_else(|| self.subscribe(connection, rpc_op));
-        match sender.send(request).await {
-            Ok(_) => Ok(()),
-            Err(_) => Err(GrpcServerError::ClosedHandler(rpc_op)),
+        match method.routing_policy() {
+            RoutingPolicy::Enqueue => match sender.send(request).await {
+                Ok(_) => Ok(()),
+                Err(_) => Err(GrpcServerError::ClosedHandler(rpc_op)),
+            },
+            RoutingPolicy::DropIfFull => match sender.try_send(request) {
+                Ok(_) => Ok(()),
+                Err(MpmcTrySendError::Full(_)) => Err(GrpcServerError::RouteIsFull(rpc_op)),
```

Comment:
I think this should not lead to a disconnection but rather returned to the client as RpcError

Reply:
actually as `SubmitBlockReport::Reject`  @tiram88

Reply:
done